docs/dev-tools/grokdebugger/index.asciidoc

```diff
@@ -1,25 +1,114 @@
 [role="xpack"]
 [[xpack-grokdebugger]]
-== Debugging Grok Expressions
+== Debugging grok expressions
 
 Grok is a pattern matching syntax that you can use to parse arbitrary text and
-structure it. Grok is perfect for parsing syslog logs, apache and other
-webserver logs, mysql logs, and in general, any log format that is generally
-written for humans and not computer consumption.
+structure it. Grok is good for parsing syslog, apache, and other
+webserver logs, mysql logs, and in general, any log format that is
+written for human consumption.
 
 Grok patterns are supported in the ingest node
 {ref}/grok-processor.html[grok processor] and the Logstash
 {logstash-ref}/plugins-filters-grok.html[grok filter]. The Elastic Stack ships
-with over 120 reusable grok patterns. See
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[Ingest node grok patterns] and https://github.com/logstash-plugins/logstash-patterns-core/tree/master/patterns[Logstash grok patterns]
+with more than 120 reusable grok patterns. See
+https://github.com/elastic/elasticsearch/tree/master/libs/grok/src/main/resources/patterns[Ingest node grok patterns] and https://github.com/logstash-plugins/logstash-patterns-core/tree/master/patterns[Logstash grok patterns]
 for the full list of patterns.
 
-You can build and debug grok patterns in the Grok Debugger tool in {kib}
+You can build and debug grok patterns in the *Grok Debugger* in {kib}
```

Comment:
I might move this sentence up to the beginning, to let the user know how the background info on Grok relates to work they're doing in Kibana